### ISSUE TITLE
Arch Linux deploy is broken

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1181]
This hangs:
```
==> Verifying source file signatures with gpg...
    quick-lint-js-3.1.0.tar.gz ... 
```

**Comment by strager** [comment]
I noticed that my GPG key used for signing is expired. However, it's been expired for 8 months, so I would have expected to see problems earlier:
```
$ gpg --list-keys -v A6851D57A65803E98C05DA01C08A7BC89CA2F557
gpg: enabled compatibility flags:
gpg: using pgp trust model
gpg: Note: signature key D6772EDE85F67FC1 expired 2023-04-21 01:24:41
pub   rsa4096 2022-04-21 [SC]
      A6851D57A65803E98C05DA01C08A7BC89CA2F557
uid           [ultimate] Matthew "strager" Glazar <strager.nds@gmail.com>
sub   rsa4096 2022-04-21 [S] [expired: 2023-04-21]
```

**Comment by strager** [comment]
Fixed in Git commit 66d8f76c0ccb5ea7da86ca7357492b1608cb076f.
